src/librustc_apfloat/ieee.rs

```diff
@@ -2310,6 +2307,17 @@ mod sig {
         limbs.iter().all(|&l| l == 0)
     }
 
+    /// One, not zero, based LSB. That is, returns 0 for a zeroed significand.
+    pub(super) fn olsb(limbs: &[Limb]) -> usize {
+        for i in 0..limbs.len() {
+            if limbs[i] != 0 {
```

Comment:
With `enumerate`? That's a transformation that can probably be done in a few places but I didn't do it in the original port.